acl/policy.go

```diff
@@ -174,7 +174,7 @@ func expandNamespacePolicy(policy string) []string {
 		NamespaceCapabilityReadScalingPolicy,
 	}
 
-	write := append(read, []string{
+	write := append(read, []string{ //nolint:gocritic
```

Comment:
We should fix this. AFAIK Go does not guarantee `read` won't be modified. I would be shocked if that ever happened, but it seems like we should fix the code instead of ignore the linter.

```suggestion
	write := make([]string, len(read))
	copy(write, read)
	write := append(write, []string{
```